src/screens/Search.js

```diff
@@ -42,6 +43,39 @@ export default function SearchScreen() {
   //   }
   // }
 
+  function showResults() {
+    if (friend.email === undefined) {
+      console.log('friend initial', friend);
+      return (
+        <Title style={{ alignSelf: 'center', marginVertical: 100 }} >There are no results to your request</Title>
+      );
+    } else {
+      return (
+        <Card elevation={10} style={{ marginVertical: 50, marginHorizontal: 10, borderWidth: 1 }} >
+          <Card.Cover source={{ uri: 'https://upload.wikimedia.org/wikipedia/commons/thumb/7/7c/User_font_awesome.svg/1200px-User_font_awesome.svg.png' }} />
```

Comment:
Long string literals should generally be stored in variables or constants to make code more legible